website/ocular-docusaurus-plugin/index.js

```diff
@@ -0,0 +1,51 @@
+// const util = require('util');
+
+/* Print full webpack error;
+   edit node_modules/react-dev-utils/formatWebpackMessages.js
+
+  ```
+  } else if ('message' in message) {
+    return message.message; 
+  ```
+*/
+
+module.exports = function (
+  context,
+  opts = {
+    resolve: {modules: [], alias: {}},
+    debug: false,
+    module: {},
+    plugins: []
+  }
+) {
+  return {
+    name: 'ocular-docusaurus-plugin',
```

Comment:
What's the connection to ocular? Is it involved in building?

Reply:
The plan is to eventually extract it to a standalone module that we can share across multiple repos.